docs/containers/quickstart-node.md

```diff
@@ -89,18 +101,29 @@ When the Docker extension adds files to the application, it also adds a **VS Cod
 
 1. Make sure the **Docker Node.js Launch** debugger configuration is selected.
 
-   ![Selected Docker debug configuration](images/quickstarts/node-debug-configuration.png)
+   ![Selected Docker debug configuration](images/quickstarts/node-debug-configuration-dark.png)
 
 1. Start debugging (use the `kb(workbench.action.debug.start)` key).
     - The Docker image for the service builds.
     - The Docker container for the service runs.
     - The browser opens to the (random) port mapped to the service container.
-    - The debugger hits the breakpoint in `index.js`.
+    - The debugger stops at the breakpoint in `index.js`.
 
     > Note that, because the debugger attaches *after* the application starts, the breakpoint may missed the first time around; you might have to refresh the browser to see the debugger break on the second try.
     >
     > You can configure the application to wait for the debugger to attach before starting execution by setting the [inspectMode](/docs/containers/reference.md#node-object-properties-docker-run-task) property to `break` in the `docker-run: debug` task in `tasks.json` under the `node` object.
 
+## Use the Docker view
+
+The Docker view provides an interactive experience to examine and manage Docker assets such as containers, images and so on. To see an example:
```

Comment:
Docker view is referenced in the "Run the service container" paragraph above. This paragraph sounds a bit as if we were assuming the user is encountering Docker view for the first time. Thus it feels a bit out of place